AmazonChimeSDK/AmazonChimeSDK/ingestion/IngestionEventConverter.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  IngestionEventConverter.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// `IngestionEventConverter` converts data from payload into `MeetingEventItem`/`DirtyEventItem`or vice versa.
+@objcMembers public class IngestionEventConverter: NSObject {
+    private let nameKey = "name"
+    private let timestampKey = "ts"
+    private let idKey = "id"
+    private let ttlKey = "ttl"
+
+    public override init() {
+        super.init()
+    }
+
+    func toIngestionMeetingEvent(event: SDKEvent, ingestionConfiguration: IngestionConfiguration) -> IngestionMeetingEvent {
+        let eventAttributes = event.eventAttributes
+        let meetingConfig = ingestionConfiguration.clientConfiguration as? MeetingEventClientConfiguration
+
+        var meetingStatus: String?
+        if let status = eventAttributes[EventAttributeName.meetingStatus] as? MeetingSessionStatusCode {
+            meetingStatus = String(describing: status)
+        }
+
+        var videoErrorStr: String?
+        if let videoError = eventAttributes[EventAttributeName.videoInputError] as? CaptureSourceError {
+            videoErrorStr = String(describing: videoError)
+        }
+
+        // Some meta data like meetingId is needed
+        // Since these meta data changes from meeting to meeting
```

Comment:
Is this still accurate?

Reply:
This is still correct if people force closes the app.